packages/template-retail-react-app/app/commerce-api/einstein.js

```diff
@@ -69,7 +69,11 @@ class EinsteinAPI {
             }
         } else if (
```

Comment:
Can you leave a TODO for refactoring the function? For the future, like when we move out this file out of the commerce-api folder.

Reply:
I'll make note of it in either a new ticket of the tick for that einstein work (save time having all this rebuild again)